packages/theme-ui/src/provider.js

```diff
@@ -32,15 +35,22 @@ const RootProvider = ({ theme = {}, components, children }) => {
   // components are provided in the default Context
   const outer = useThemeUI()
   const [colorMode, setColorMode] = useColorState(theme.initialColorMode)
+  const [themeState, setThemeState] = useReducer(mergeState, theme)
```

Comment:
This is fine for now – but I'll add the state and window object to the ThemeProvider in a separate PR – so eventually you can probably undo any changes I started in this branch